pkg/controller/hostpathprovisioner/storagepool.go

```diff
@@ -349,7 +351,7 @@ func (r *ReconcileHostPathProvisioner) storagePoolDeploymentByNode(logger logr.L
 		Spec: appsv1.DeploymentSpec{
 			Selector: &v1.LabelSelector{
 				MatchLabels: map[string]string{
-					"hpp-pool": getResourceNameWithMaxLength(sourceStoragePool.Name, "hpp", maxNameLength),
+					"hpp-pool": resourceName,
```

Comment:
maybe put "hpp-pool" as a constant, we are using it in multiple places in this file